WalletWasabi.Documentation/Guides/UpgradeTorDaemons.md

```diff
@@ -1,5 +1,19 @@
 ## How to upgrade bundled Tor version in Wallet Wasabi
 
+### Automatically
+
+Invoke the following commands in your PowerShell 7+:
+
+```powershell
+cd WalletWasabi/Microservices/Binaries
+
+# See what the latest released Tor Browser version is here: https://dist.torproject.org/torbrowser/.
```

Comment:
Currently https://dist.torproject.org/torbrowser/ contains a `11.5.3` folder but it only contains binaries for android, so IMO we should use https://www.torproject.org/download/ to check the latest released Tor Browser version.

```suggestion
# See what the latest released Tor Browser version is here: https://www.torproject.org/download/
```